src/BlockHorizons/BlockPets/listeners/EventListener.php

```diff
@@ -119,17 +118,6 @@ public function onChat(PlayerChatEvent $event): void {
 		}
 	}
 
-	public function onEntitySpawn(EntitySpawnEvent $event): void {
-		$entity = $event->getEntity();
-		if($entity instanceof BasePet) {
-			$clearLaggPlugin = $this->getLoader()->getServer()->getPluginManager()->getPlugin("ClearLagg");
-			if($clearLaggPlugin !== null) {
```

Comment:
This plugin is still actively updated, should we be removing this functionality?